jest.config.js

```diff
@@ -44,8 +44,8 @@ module.exports = {
   // We have to specify the environment we are running in, which is jsdom. The
   // default is 'node'. This can be modified *per file* using a comment at the
   // head of the file. So it may be worthwhile to switch to 'node' in any
-  // background tests.
-  testEnvironment: 'jsdom',
+  // background tests. `jest-fixed-jsdom` is an improved version of jsdom.
+  testEnvironment: 'jest-fixed-jsdom',
```

Comment:
`fake-indexeddb` needs the `structuredClone` polyfill, since it is only used in a Node.js-based unit test. The `jsdom` environment doesn't include `structuredClone` (which has been available in all our supported browsers since `2022-02-01`).

Reply:
Interesting. The README for `jest-fixed-jsdom` is giving some mixed messages - the motivation section seems clear enough, but I don't know what they mean by the warning about it being a workaround not a fix/solution. Maybe that's just a plea for this to get fixed upstream?

Reply:
Jest's JSDom environment is missing a bunch of newer APIs that node.js actually has built-in.

I'm using `jest-fixed-jsdom` here just for this line: https://github.com/mswjs/jest-fixed-jsdom/blob/6c015b39f381cd80a7621d1e41fe263f6c7398b8/index.js#L30

Maybe the package I updated will fine if I add `structuredClone` to `global` in our `setup-helper.js` (I'm not sure it has a `node:` package though, so it might be hard to patch back in)? I'm not really sure how this all works 😅